ports/glutin/window.rs

```diff
@@ -702,6 +702,12 @@ impl WindowMethods for Window {
             (NONE, Key::Down) => {
                 self.scroll_window(0.0, -3.0 * LINE_HEIGHT);
             }
+            (NONE, Key::Left) => {
+                self.scroll_window(LINE_HEIGHT, 0.0);
+            }
+            (NONE, Key::Right) => {
+                self.scroll_window(-LINE_HEIGHT, 0.0);
```

Comment:
I think it's preferable for the horizontal scroll to be more fine grained than the vertical scroll, by default.